Sources/ObjCSupport/LOKStackLayout.swift

```diff
@@ -41,6 +41,8 @@ extension LOKStackLayoutDistribution {
 }
 
 @objc open class LOKStackLayout: LOKBaseLayout {
+    @objc public let sublayouts: [LOKLayout]?
```

Comment:
Should we expose the other attributes of these layouts as well? Axis and spacing for example. Or is there something special about these properties that makes them useful and necessary?

Reply:
There are computed vars for`flexibility` and `viewReuseId` in `LOKBaseLayout`, so added properties for all remaining layout parameters.